account_invoice_pricelist/README.rst

```diff
@@ -0,0 +1,62 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
Prefer the png image

Reply:
Or even better, use the README by fragments for letting the bot to generate this

Reply:
Please use png